axelrod/fingerprint.py

```diff
@@ -0,0 +1,237 @@
+import axelrod as axl
+import numpy as np
+# import matplotlib.pyplot as plt
+from axelrod.strategy_transformers import JossAnnTransformer, DualTransformer
+from axelrod.interaction_utils import compute_final_score_per_turn as cfspt
+from collections import namedtuple
+
+
+def create_jossann(coordinate, probe):
+    """Creates a JossAnn probe player that matches the coordinate.
+
+    If the coordinate sums to more than 1 the parameters are flipped and
+    subtracted from 1 to give meaningful probabilities. This is outlined further
+    in [Ashlock2010]_.
+
+    Parameters
+    ----------
+    coordinate : tuple of length 2
+        coordinate of the form (x, y)
+    probe : class
+        A class that must be descended from axelrod.strategies
+
+    Returns
+    ----------
+    joss_ann: Joss-AnnTitForTat object
+        `JossAnnTransformer` with parameters that correspond to (x, y).
+    """
+    x, y = coordinate
+    if x + y > 1:
+        joss_ann = JossAnnTransformer((1 - y, 1 - x))(probe)()
+    else:
+        joss_ann = JossAnnTransformer((x, y))(probe)()
+    return joss_ann
+
+
+def create_coordinates(step):
+    """Creates a set of coordinates over the unit square.
+
+    Constructs (x, y) coordinates that are separated by a step equal to
+    `step`. The coordinates are over the unit squeare which implies
+    that the number of points created will be 1/`step`^2.
+
+    Parameters
+    ----------
+    step : float
+        The separation between each coordinate. Smaller steps will
+        produce more coordinates that will be closer together.
+
+    Returns
+    ----------
+    coordinates : list of tuples
+        Tuples of length 2 representing each coordinate, eg. (x, y)
+    """
+    Point = namedtuple('Point', 'x y')
```

Comment:
This could be defined at the module level and then used throughout instead of having a mixture of named and anonymous tuples.